apps/daimo-mobile/src/view/TabNav.tsx

```diff
@@ -0,0 +1,186 @@
+import { assertUnreachable } from "@daimo/common";
+import { chainConfig } from "@daimo/contract";
+import Octicons from "@expo/vector-icons/Octicons";
+import {
+  BottomTabNavigationOptions,
+  createBottomTabNavigator,
+} from "@react-navigation/bottom-tabs";
+import { RouteProp } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import { useEffect, useMemo, useState } from "react";
+
+import { AddDeviceScreen } from "./screen/AddDeviceScreen";
+import { DeviceScreen } from "./screen/DeviceScreen";
+import { HistoryOpScreen } from "./screen/HistoryOpScreen";
+import { HistoryScreen } from "./screen/HistoryScreen";
+import HomeScreen from "./screen/HomeScreen";
+import OnboardingScreen from "./screen/OnboardingScreen";
+import { SettingsScreen } from "./screen/SettingsScreen";
+import NoteScreen from "./screen/link/NoteScreen";
+import CreateRequestScreen from "./screen/receive/CreateRequestScreen";
+import DepositScreen from "./screen/receive/DepositScreen";
+import SendRequestScreen from "./screen/receive/SendRequestScreen";
+import SendScreen from "./screen/send/SendScreen";
+import { OctName } from "./shared/InputBig";
+import {
+  ParamListHome,
+  ParamListReceive,
+  ParamListSend,
+  ParamListSettings,
+  ParamListTab,
+} from "./shared/nav";
+import { color } from "./shared/style";
+import { useAccount } from "../model/account";
+
+const Tab = createBottomTabNavigator<ParamListTab>();
+
+export function TabNav() {
+  return (
+    <Tab.Navigator initialRouteName="HomeTab" screenOptions={getTabOptions}>
+      <Tab.Screen name="DepositTab" component={DepositTab} />
+      <Tab.Screen name="SendTab" component={SendTab} />
+      <Tab.Screen name="HomeTab" component={HomeTab} />
+      <Tab.Screen name="ReceiveTab" component={ReceiveTab} />
+      <Tab.Screen name="SettingsTab" component={SettingsTab} />
+    </Tab.Navigator>
+  );
+}
+
+function getTabOptions({
+  route,
+}: {
+  route: RouteProp<ParamListTab, keyof ParamListTab>;
+}): BottomTabNavigationOptions {
+  const all: BottomTabNavigationOptions = {
+    headerShown: false,
+    tabBarStyle: {
+      height: 72,
+      paddingTop: 16,
+      paddingHorizontal: 16,
+    },
+    tabBarItemStyle: {
+      height: 44,
+    },
+  };
+  const head = { ...all, headerShown: true };
```

Comment:
did you mean to hide the header?

Reply:
yes, some tabs need a header others not (otherwise they get a double, nested header)